coalib/output/Logging.py

```diff
@@ -36,14 +88,22 @@ def configure_logging():
                     'INFO': 'blue',
                     'DEBUG': 'green'
                 }
+            },
+            'no-color': {
+                '()': 'logging.Formatter',
+                'format': '[%(levelname)s][%(asctime)s] %(message)s'
```

Comment:
okay for this PR, though I think we should define this format string globally in this module so we can reuse it :+1: